Changed burgers from div to button. Fixed missing hover

chakler opened the pull request:
I think using button for the burgers might be more semantic then using divs. :smiley:
Also added a pseudo element box to some of the burgers so that it'd still be easy to hover even if it's in active state.

nkpgardose commented:
@chakler yes it is semantic. I'm also thinking about changing it to button. However, when applying bootstrap, skeletoncss or other styling. Burgers breaks and that's why I'm sticking to div for burger component.

I like the pseudo for hover! :+1:

chakler commented:
@nkpgardose Ops didn't think of that :stuck_out_tongue: I've changed the buttons back to divs and added `role="button"` and `tabindex="0"` to them. They still aren't buttons but at least they act like one now :smiley:

nkpgardose commented:
This is great! :hamburger: 
![drake_approves](https://cloud.githubusercontent.com/assets/3381818/8631886/7b31b7cc-27b6-11e5-9d93-dbd4f0345a1a.png)
